processing/src/main/java/io/druid/query/extraction/StringFormatExtractionFn.java

```diff
@@ -32,15 +33,47 @@
  */
 public class StringFormatExtractionFn extends DimExtractionFn
 {
+  public static enum NullHandling
+  {
+    NULLSTRING,
```

Comment:
I assume the idea here is to make it easy for `NullHandling.valueOf(value.toUpperCase())` to work on "nullString"